scripts/presubmit.sh

```diff
@@ -77,6 +77,14 @@ main() {
     grep -v asn1/ | \
     tr '\n' ' ')"
 
+  # Prevent the creation of proto files with .txt extensions.
+  bad_protos="$(find . -name '*.pb.txt' -o -name '*.proto.txt')"
```

Comment:
nit: Add `-print` to separate each file path with a line break, rather than just a space. This makes the output below a little easier to read when there are multiple files.

Reply:
Can you make the same changes to the script in the trillian repo?

Reply:
Done.